xoinvader/background.py

```diff
@@ -0,0 +1,261 @@
+from xoinvader.common import Settings
+from xoinvader.utils import Point, Surface
+from xoinvader.render import Renderable
+
+
+class Chunk(object):
+    """Class for storing background chunks.
+
+    :param name: name of the chunk
+    :type name: string
+
+    :param lines: list of all chunk lines
+    :type lines: list
+
+    :param length: length of lines list (for convenience)
+    :type length: integer
+
+    :param __MAGIC__: magic of chunk boundary in the file
+    :type __MAGIC__: string
+    """
+
+    __MAGIC__ = "~chunk~"
```

Comment:
You are accessing this from outer scope, please move out to top of file as constant.
Name it like you think is right.

```python
CHUNK_MAGIC = "~chunk~"
"""Magic of chunk boundary in the file."""
```